### PR TITLE
add backwards compatibility for unstructured tensor search

### PR DESCRIPTION
**Pull request by vicilliar**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
bug fix

* **What is the current behavior?** (You can also link to an open issue here)
tensor search backwards compatibility is broken in unstructured indexes.

* **What is the new behavior (if this is a feature change)?**
fixed query input parameter used for embedding

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Have unit tests been run against this PR?** (Has there also been any additional testing?)
in progress

* **Related Python client changes** (link commit/PR here)

* **Related documentation changes** (link commit/PR here)

* **Other information**:

* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py [hunk 1]
```diff
@@ -318,9 +323,14 @@ def _to_vespa_hybrid_query(self, marqo_query: MarqoHybridQuery) -> Dict[str, Any
         summary = unstructured_common.SUMMARY_ALL_VECTOR if marqo_query.expose_facets \
             else unstructured_common.SUMMARY_ALL_NON_VECTOR
 
+        if self._marqo_index_version >= self._HYBRID_SEARCH_MINIMUM_VERSION:
```
Comment (farshidz) [inline src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py]: We don't support hybrid for these versions to why this branching?

Comment (vicilliar) [reply src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py]: fixed

File: src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py [hunk 2]
```diff
@@ -113,14 +118,19 @@ def _get_tensor_search_term(marqo_query: MarqoTensorQuery) -> str:
             target_hits = marqo_query.limit + marqo_query.offset
             additional_hits = 0
 
+        if self._marqo_index_version >= self._HYBRID_SEARCH_MINIMUM_VERSION:
```
Comment (farshidz) [inline src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py:120]: Calling self in a static method

Comment (vicilliar) [reply src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py:120]: fixed

**Review by farshidz — APPROVED** [review]

**Pull request merged**